### ISSUE TITLE
Componente `buttons` - Verificare colori e focus

### ISSUE THREAD

**Issue by Fupete** [italia/design-angular-kit#364]
## Cosa

Verificare le varianti e i colori. 
Verificare l'implementazione corretta dello stato focus, ad esempio al momento non presente nella variante base senza colore e con un comportamento diverso da quello in Bootstrap Italia o UI Kit Italia. 

## Perché
Coerenza con tutte le risorse del design system .italia. 

## Contesto
Verifiche di accessibilità sui componenti.

Sarà da aggiornare lo status delle verifiche nella [board census ](https://github.com/orgs/italia/projects/32/views/1)alla risoluzione.

## Altro

**Comment by AntoninoBonanno** [comment]
Per il non colore ti basta non passarlo 

```<button itButton>Link</button>```
```<button [itButton]="undefined">Link</button>```

è l'equivalente di

```<button type="button" class="btn">Link</button>```

Per le altre varianti mi sembrano ci siano tutte.
Per la variante delle icone se inserisci un `ItIconComponent` all'interno del button verrà aggiunto in automatico la classe `btn-icon`

**Comment by astagi** [comment]
Grazie @Fupete @AntoninoBonanno

**Comment by Fupete** [comment]
Ad un primo check la segnalazione rimane aperta.  

Ad esempio per il focus (da tastiera...) la variante senza stile (solo link) non ha focus evidente nella documentazione. 
I focus delle altre varianti non seguono quanto negli altri framework (un colore solo e non ogni variante il suo colore). 

Seppur cambieremo l'approccio del focus nel design system appena possibile (doppia riga nera e bianca probabilmente per essere ancora più accessibili), intanto sarebbe bene avere un unico approccio uniforme tra tutti i tre framework Bootstrap Italia, React e Angular kit. Uniformando Angular agli atlri due.

**Comment by astagi** [comment]
Per ora ho chiuso questa issue con la PR #408 se ci sono altri problemi apriamo issue ad hoc.
